src/OrchardCore.Build/Dependencies.AspNetCore.props

```diff
@@ -1,43 +1,55 @@
 <Project>
 
+  <!-- Default TFMs to use when building and packaging -->
   <PropertyGroup>
-    <DefaultTargetFramework>net5.0</DefaultTargetFramework>
-    <AspNetCoreTargetFrameworks Condition="'$(AspNetCoreTargetFrameworks)' == ''">net5.0;netcoreapp3.1</AspNetCoreTargetFrameworks>
+
+    <!-- TFM used when coding in Visual Studio and when creating templates -->
+    <DefaultTargetFramework Condition="'$(DefaultTargetFramework)' == ''">net6.0</DefaultTargetFramework>
+    
+    <!-- TFMs used to build the abstractions and modules-->
+    <CommonTargetFrameworks Condition="'$(CommonTargetFrameworks)' == ''">$(DefaultTargetFramework)</CommonTargetFrameworks>
```

Comment:
So, if we do this here, what we do below line 15 is useless

Before `DefaultTargetFramework` was just used as an inner variable, that's why it was set without any condition

And was just used to set the TFMs if e.g. under VStudio